Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -178,60 +180,53 @@ int RGroupDecomposition::getMatchingCoreInternal(
     size_t i = 0;
     for (const auto &mv : tmatches) {
       bool passes_filter = data->params.onlyMatchAtRGroups;
-      boost::dynamic_bitset<> target_match_indices(mol.getNumAtoms());
+      std::vector<std::vector<int>> targetToCoreIndices(mol.getNumAtoms());
```

Comment:
This data structure is used quite a bit now, we should document what it is doing.  It appears to be mapping

atomIndex -> vector of possible matches?

Reply:
Good point, I added a detailed explanation as it is not obvious.